templates/web/base/report/form/user_loggedout_password.html

```diff
@@ -1,6 +1,6 @@
-<div id="form_sign_in_yes" class="form-box">
+<div id="form_sign_in_yes" class="form-box hidden-js js-new-report-sign-in-shown">
 
-    <h5>[% loc('<strong>Yes</strong> I have a password') %]</h5>
+    <h5 class="hidden-js">[% loc('<strong>Yes</strong> I have a password') %]</h5>
 
     <label for="password_sign_in">[% loc('Your password') %]</label>
     [% IF field_errors.password %]
```

Comment:
If you submit a report with a password and you got the password wrong then it's very confusing as you get shown the main report form so to see the error you have to click "Continue" and then the show pre-fill link. There probably need to be a way to skip over that and directly display the password form.